### ISSUE TITLE
update to Scala 2.10

### ISSUE THREAD

**Issue by medovina** [ornicar/scalex#43]
Please update Scalex to search the libraries for Scala 2.10.  Thanks!

**Comment by Blaisorblade** [comment]
:+1:
And what about upgrading Scalaz to 7?

**Comment by ornicar** [comment]
Yep, work in progress!

**Comment by coltfred** [comment]
+1

Scala 2.10 and Scalaz 7 would be awesome!

**Comment by MasseGuillaume** [comment]
Hi @ornicar any progress yet?

**Comment by ornicar** [comment]
The project is in standby at the moment. I'm busy with http://lichess.org and http://river.scalex.org.

Maybe scala.io will motivate me to get back on scalex.

**Comment by MasseGuillaume** [comment]
We are working on a project called codebrew (see http://vimeo.com/75362996, the doc part at 2:41). 

It's 100% Open-source (https://github.com/CodeBrew-io). I hope this motivate you even more :)

**Comment by MasseGuillaume** [comment]
@ornicar I can migrate your project to 2.10/2.11

I fixed sbt dependency / build problems, it would be pretty cool if you can give me 30 minute of your time to get the whole picture via video chat.

**Comment by ornicar** [comment]
This won't make it support more scala versions.

https://twitter.com/ornicar/status/398002433671979011

To speed things up, I'll release the next scalex with only keyword search support. Type search is the hard part, and will have to wait a bit more.

**Comment by MasseGuillaume** [comment]
perfect thanks alot :-)

**Comment by jedesah** [comment]
Hey @ornicar,

Just wanted to let you know that we (codebrew.io) will be doing somewhat of a formal release in just over two weeks. As I think you are already aware, we are using scalex, but it's not giving the best results because we evaluate code using Scala 2.10 but our search bar returns results for 2.9 (the lack of example code in the 2.9 docs is the biggest annoyance in my opinion).

Of course it's not a show stopper and if we really wanted it fixed no matter what, we should just do it ourselves (so many other things to do though). So if this knowledge can help in any way to motivate you to put a little extra effort into Scalex and this issue in particular then of course that would be awesome.

In any case we would like to add a small "powered by scalex" somewhere near the search bar to give credit where credit is due, if that's okay with you of course. Let us know if you have any suggestions on that front.

**Comment by ornicar** [comment]
Hello, you're right to remind me that my work on this project matters. I also have many other things to do (prismic.io by day, lichess.org by night) but I'll do my best to get scalex 3 working within 2 weeks.

**Comment by jedesah** [comment]
Awesome! Don't outwork yourself though, you've obviously got a lot on on your plate and we totally understand that.

**Comment by laughedelic** [comment]
Any updates here?

**Comment by ornicar** [comment]
No, I'm afraid i stopped working on it. Just too busy with prismic.io and lichess.org.

**Comment by erikkaplun** [comment]
Is the project currently abandoned?
